utils/use_require.js

```diff
@@ -39,15 +39,14 @@ const no_transform_files = new Set([
     path.join(paths.app, 'error-handler.js'),
 ]);
 
-no_copy_files.forEach((file) => no_transform_files.add(file));
+no_copy_files.forEach(file => no_transform_files.add(file));
 
 // util.promisify requires Node.js 8.x, so we have our own
 function promisify(original) {
-    return function () {
-        const obj = this;
+    return () => {
         const args = Array.prototype.slice.call(arguments);
         return new Promise((resolve, reject) => {
-            original.apply(obj, args.concat((err, value) => {
+            original.apply(this, args.concat((err, value) => {
```

Comment:
This will be incorrect. `this` needs to be from the call to the unnamed function. With an arrow function you'll instead get `this` from the `promisify()` call.